src/main/java/com/ontotext/kafka/service/SinkRecordsProcessor.java

```diff
@@ -140,8 +141,9 @@ public Object call() throws RetriableException {
 			}
 			connection.commit();
 			failedRecords.clear();
+			long finish = System.currentTimeMillis();
+			LOGGER.trace("Finished batch processing for {} ms", finish - start);
 			return SUCCESSES;
-
 		} catch (RepositoryException e) {
```

Comment:
by the way we can change this to general Exception (even though not the best practice) as we are not handling any runtime exceptions (e.g. aborted, timeout)